python/test_file_generator.py

```diff
@@ -1321,7 +1240,7 @@ def v654_1_file(self):
 		path = self.validity_path
 		ts = self.ts_df.copy()
 		lar = self.lar_df.copy()
-		lar.income = lar.income.map(lambda x: random.choice(("1.5", "")))
+		lar.income = lar.income.apply(lambda x: "1.5")
```

Comment:
I took out the option for blanks here, but there are still formatting errors rather than a v654 edit fail when income is filled in with a decimal value.

Reply:
The Parser is likely checking data type. In this case integer vs float.